common/thorhelper/thorstats.cpp

```diff
@@ -20,34 +20,63 @@
 #include "thorstats.hpp"
 #include "jdebug.hpp"
 
+// Available sets of statistics for a nested section
+const StatisticsMapping defaultNestedSectionStatistics({StCycleLocalExecuteCycles, StTimeLocalExecute, StNumStarts, StNumStops});
+const StatisticsMapping genericCacheNestedSectionStatistics({StCycleLocalExecuteCycles, StTimeLocalExecute, StNumStarts, StNumStops, StNumCacheAdds, StNumCacheHits, StNumPeakCacheObjects});
```

Comment:
genericCacheNestedSectionStatistics should build on defaultNestedSectionStatistics so that if defaults are ever changed, the changes are reflected in genericCacheNestedSectionStatistics:

const StatisticsMapping genericCacheNestedSectionStatistics(defaultNestedSectionStatistics, {StNumCacheAdds, StNumCacheHits, StNumPeakCacheObjects})
```suggestion
const StatisticsMapping genericCacheNestedSectionStatistics({StCycleLocalExecuteCycles, StTimeLocalExecute, StNumStarts, StNumStops, StNumCacheAdds, StNumCacheHits, StNumPeakCacheObjects});
```

Reply:
Fixed. Note that defaultNestedSectionStatistics is actually the last arg to the next constructor rather than the first.